osu.Game/Beatmaps/Formats/Decoder.cs

```diff
@@ -37,6 +38,15 @@ static Decoder()
             LegacyStoryboardDecoder.Register();
         }
 
+        /// <summary>
+        /// Register dependencies for use with static decoder classes.
+        /// </summary>
+        /// <param name="rulesets">A store containing all available rulesets (used by <see cref="LegacyBeatmapDecoder"/>).</param>
+        public static void RegisterDependencies(RulesetStore rulesets)
```

Comment:
I've left this in the base decoder class in case there are other dependencies to be registered in the future that may be used by different decoders. It could optionally be moved local to `LegacyBeatmapDecoder` if that's preferred.